tests/uat/uat_sanity_suite_test.go

```diff
@@ -201,3 +201,17 @@ func TestVirtualDevice2PortCreateConnection_PNFV(t *testing.T) {
 	output := terraform.Output(t, terraformOptions, "port_connection_id")
 	assert.NotNil(t, output)
 }
+
+func TestCloudRouter2VirtualDeviceCreateConnection_PFCR(t *testing.T) {
```

Comment:
You've added this test to the PNFV Test user in GHA, but here the user is specified as PFCR. Which one is the correct one?

Reply:
Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/111/commits/82795304554522209684b3df70ee93fcfbf1979e) commit